packages/sampler/stories/qa/Popup.js

```diff
@@ -30,6 +30,59 @@ const PopupFromSelfOnlyContainer = Toggleable(
 	)
 );
 
+class PopupResumeFocusAfterOpenState extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			isPopup: false,
+			isNotification: false
+		};
+	}
+
+	handlePopup = () => {
+		this.setState({
+			isPopup: true
+		});
+
+		setTimeout(() => {
+			this.setState({
+				isPopup: false
+			});
+		}, 200);
+	}
+
+	handleClose = () => {
+		this.setState({
+			isPopup: false
+		});
+	}
+
+	handleNotification = () => {
+		this.setState({
+			isNotification: false
+		});
+	}
+
+	render () {
+		return (
+			<div>
+				<Button onClick={this.handlePopup}>open</Button>
+				<Button>hello</Button>
```

Comment:
Is this button supposed to open the popup?  If so, let's rename the buttons to clarify which does what.  Otherwise, there's no way to open the `Popup`

Reply:
It was based on ENYO-6074. The Popup opens quickly and closes quickly. Then we switch to 5-way and 5-way Spot 'hello'.